src/app/cli/src/commands/system_api_server_run_command.rs

```diff
@@ -155,13 +154,17 @@ impl Command for APIServerRunCommand {
                 s(format!("http://{}", api_server.local_addr())).bold(),
             );
             eprintln!("{}", s("Use Ctrl+C to stop the server").yellow());
+        }
 
-            if self.get_token {
+        if self.get_token {
+            if self.output_config.is_tty && self.output_config.verbosity_level == 0 {
```

Comment:
> self.output_config.is_tty && self.output_config.verbosity_level == 0

Minor: let's make this a separate method

PS. I don't fully understand why we don't use `self.output_config.verbosity_level > 0` instead.

Reply:
We have #318 to improve this mess - for now I stay on the side of consistency with other code.